src/frontend/screens/Settings/components/Tools/index.tsx

```diff
@@ -131,12 +144,15 @@ export default function Tools() {
           <a
             onDrop={async (ev) => dropHandler(ev)}
             onDragOver={(ev) => dragOverHandler(ev)}
-            className="button outline drag"
+            className={classNames('button outline drag', {
+              active: runExeRunning
+            })}
             onClick={handleRunExe}
           >
-            {t('setting.runexe.title')}
-            <br />
-            <span>{t('setting.runexe.message')}</span>
+            <div>
+              {t('setting.runexe.title')} <br />
+              <span>{t('setting.runexe.message')}</span>
+            </div>
```

Comment:
we don't need this div, we can also remove the new css, we need to add a `justify-content: center` to `.drag` fix the alignment of the run exe on prefix button

Reply:
fixed as well